frontend/src/metabase/ui/utils/colors.ts

```diff
@@ -22,8 +22,6 @@ const ORIGINAL_COLORS = [
 ];
 
 const CUSTOM_COLORS = [
-  "accent4",
-  "accent5",
```

Comment:
Simplify the type. If you see the history, these 2 colors are added just to be used only once, so I inline `theme.fn.themeColor` for them instead.